### PR TITLE
Last updates on download / setup / NB example to official release

### PR DESCRIPTION
**Pull request by andrelmfarias**

### CODE REVIEW

File: download.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import os
+import wget
+import requests
+from github import Github
```
Comment (fmikaelian) [inline download.py]: We can remove this line and the `pyGithub` repo from requirements?

File: download.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        wget.download(url=squad_url, out=directory)
+
+def download_models():
+    directory = 'models'
+    models_url = [
+    'https://github.com/cdqa-suite/cdQA/releases/download/bert_qa_vGPU/bert_qa_vGPU-sklearn.joblib',
+    'https://github.com/cdqa-suite/cdQA/releases/download/bert_qa_vCPU/bert_qa_vCPU-sklearn.joblib'
+    ]
+
+    print('\nDownloading trained models...')
```
Comment (fmikaelian) [inline download.py]: `\n`? haha

File: download.py (hunk 1 above)
Comment (andrelmfarias) [reply download.py]: yes, well seen

File: download.py (hunk 2 above)
Comment (andrelmfarias) [reply download.py]: Yes, we need to jump a line, if we don't we will have this: 

image

Comment (fmikaelian) [reply download.py]: Oh i see now sorry 😜

**Pull request merged**